src/task_9/task_9_3.py

```diff
@@ -0,0 +1,42 @@
+"""
+Создать декоратор для функции, которая принимает список чисел.
+Декоратор должен производить предварительную проверку данных -
+удалять все четные элементы из списка.
+"""
+
+
+def decor_even(func):
+    """
+    декоратор функции.
+    принимает на вход функцию, принимающую на вход список,
+    оставляет в списке только четные по индексу элементы
+    :param func: функция по списку из чисел
+    :return other func: функция по списку из чисел
+    (только четные индексы)
+    """
+
+    def is_even(x: list):
+        lis = [i for i in x if x.index(i) % 2 == 0]
```

Comment:
Удалить все четные :)
и имелось в виду четные не по индексу, а по значению

```suggestion
        lis = [i for i in x if i % 2 != 0]
```